.github/workflows/build_and_test.yml

```diff
@@ -133,20 +136,20 @@ jobs:
       - name: Test Tutorials
         run: cd build_examples/tutorials_bin && ./Tutorial-HelloWorld
 
-      # - name: Create Cobertura Report
-      #   run: |
-      #     ${{ matrix.python }} -m pip install gcovr
-      #     gcovr -v -r . ${{ matrix.additional-gcov-flags }} $GCOVR_FLAGS -o coverage.xml
-
-      # - name: Upload Coverage Results
-      #   uses: codecov/codecov-action@ab904c41d6ece82784817410c45d8b8c02684457 # v3.1.6
-      #   with:
-      #     files: ./coverage.xml
-      #     flags: ${{ matrix.image }},unittest
-      #     fail_ci_if_error: false
-      #     verbose: true
-      #   env:
-      #     CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      - name: Create Cobertura Report
+        run: |
+          ${{ matrix.python }} -m pip install gcovr
+          gcovr -v -d -r . ${{ matrix.additional-gcov-flags }} $GCOVR_FLAGS -o coverage.xml
```

Comment:
@seladb It was actually to remove generated coverage files to prevent misbehavior of ccache but later i found the ignore rule from extension which is a better solution. Flag is unnecessary now but I forgot to remove the flag sorry, removing now

Reply:
fixed with a9e5b6d